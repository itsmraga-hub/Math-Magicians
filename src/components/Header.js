import { Link } from 'react-router-dom';

import style from './Header.module.css';

const Header = () => (
  <header className={style.headerContainer}>
    <a href="###" className={style.logo}>Math Magicians</a>
    <ul className={style.navLinksContainer}>
      <li className={style.navLink}><Link to="/">Home</Link></li>
      <li className={style.navLink}><Link to="/calculator">Calculator</Link></li>
      <li className={style.navLink}><Link to="/quote">Quote</Link></li>
    </ul>
  </header>
);

export default Header;
