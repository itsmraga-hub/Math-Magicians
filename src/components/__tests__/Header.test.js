import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../Header';

it('<Header /> Renders properly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
