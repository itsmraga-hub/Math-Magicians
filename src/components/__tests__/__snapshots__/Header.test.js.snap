// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Renders properly 1`] = `
<header
  className="headerContainer"
>
  <a
    className="logo"
    href="###"
  >
    Math Magicians
  </a>
  <ul
    className="navLinksContainer"
  >
    <li
      className="navLink"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="navLink"
    >
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="navLink"
    >
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</header>
`;
